lib/shopify-cli/commands/help.rb

```diff
@@ -49,7 +49,7 @@ def display_help(klass)
           output += "\n"
           output += klass.extended_help
         end
-        @ctx.page(output)
+        @ctx.puts(output)
```

Comment:
`page` has different behaviours based on terminal size which was causing my tests to be flaky between local/CI.
The `help` command doesnt really need to be paged, so I'm updating this to be a `puts` instead.